prow/config.yaml

```diff
@@ -268,6 +268,12 @@ presets:
     env:
       - name: KYMA_DEVELOPMENT_ARTIFACTS_BUCKET
         value: "gs://kyma-development-artifacts"
+  - labels:
+     preset-kyma-encryption-key: "true"
+    env:
+      - name: KYMA_ENCRYPTION_KEY
+        value: projects/kyma-project/locations/global/keyRings/kyma-prow/cryptoKeys/kyma-prow-encryption\
```

Comment:
The trailing `\` here looks a bit suspicious, is it correct?

Reply:
yes its correct, believe it gets stored with the trailing
➜  ~ export KYMA_ENCRYPTION_KEY=projects/kyma-project/locations/global/keyRings/kyma-prow/cryptoKeys/kyma-prow-encryption\

➜  ~ echo $KYMA_ENCRYPTION_KEY
projects/kyma-project/locations/global/keyRings/kyma-prow/cryptoKeys/kyma-prow-encryption

I will remove it in the code